cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Functions.scala

```diff
@@ -119,6 +119,7 @@ case class Interval(unit: TimeUnit.Value, x: Int = 1) extends GroupFunc with Sca
       case Week => x * 60 * 60 * 24 * 7
       case Month => x * 60 * 60 * 24 * 30
       case Year => x * 60 * 60 * 24 * 365
+      case Date => x * 60 * 60 * 24  // MySQL: date function would return dateTime object
```

Comment:
but it is exactly what we want right? this unit is used to quantify a time range, e.g., how many days, month.... we can't not see how many date, right?